// Copyright 2014 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
//
// Summary of client variations from experiments.

syntax = "proto2";

option optimize_for = LITE_RUNTIME;

package variations;

message ClientVariations {
  // A list of client experiment variation IDs that are active.
  repeated int32 variation_id = 1;

  // A list of client experiment variation IDs that trigger server side
  // behavior.
  repeated int32 trigger_variation_id = 3;
}
